### ISSUE TITLE
List Documents Fails Querying Attribute with Dash

### ISSUE THREAD

**Issue by stnguyen90** [appwrite/appwrite#3600]
### 👟 Reproduction steps

1. Create a collection with an attribute that has a dash, like `e-mail`
2. Create a document in the collection
3. Make a `listDocuments()` API call with a query on the attribute with the dash.

### 👍 Expected behavior

The document is returned

### 👎 Actual Behavior

API results in 500 error:

```json
{
    "message": "Server Error",
    "code": 500,
    "type": "general_unknown",
    "version": "0.15.2"
}
```

Appwrite logs:

```
appwrite  | [Error] Timestamp: 2022-07-25T20:00:36+00:00
appwrite  | [Error] Method: GET
appwrite  | [Error] URL: /v1/databases/:databaseId/collections/:collectionId/documents
appwrite  | [Error] Type: TypeError
appwrite  | [Error] Message: Exception::__construct(): Argument #1 ($message) must be of type string, null given
appwrite  | [Error] File: @swoole-src/library/core/Database/PDOStatementProxy.php
appwrite  | [Error] Line: 64
```

Discord thread:

![image](https://user-images.githubusercontent.com/67883383/180656180-f80fff30-56df-4382-8310-b39806288a61.png)
here is the full Converation around this issue, creating new Attribute as "email" instead of "e-mail" fixes that.

_Originally posted by @lgecass in https://github.com/appwrite/appwrite/issues/3044#issuecomment-1193349328_

### 🎲 Appwrite version

Version 0.15.x

### 💻 Operating system

MacOS

### 🧱 Your Environment

Tested on 0.15.2.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by Meldiron** [comment]
I can confirm the same problem is there in 0.15.3. This PR might be related: https://github.com/utopia-php/database/pull/133

(PR was merged, but not released)

**Comment by Meldiron** [comment]
Just tried it with the latest utopia/database release, and it resolves the problem. The bug fix will be released with next Appwrite version 🥳

<img width="510" alt="CleanShot 2022-08-20 at 09 01 32@2x" src="https://user-images.githubusercontent.com/19310830/185733381-abffa575-5808-4fcf-848a-e66f1e60e8d6.png">

**Comment by TorstenDittmann** [comment]
This got implemented and released with Appwrite 1.0.0 🥳 

Thank you for your contribution 🙏🏻  You can read more [here](https://appwrite.io/1.0).

**Comment by stnguyen90** [comment]
I'm still seeing the problem on 1.0.2:

Response:

```json
{
    "message": "Server Error",
    "code": 500,
    "type": "general_unknown",
    "version": "1.0.1"
}
```

Logs:

```
appwrite                        | [Error] Timestamp: 2022-10-07T15:36:11+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /v1/databases/:databaseId/collections/:collectionId/documents
appwrite                        | [Error] Type: TypeError
appwrite                        | [Error] Message: Exception::__construct(): Argument #1 ($message) must be of type string, null given
appwrite                        | [Error] File: @swoole-src/library/core/Database/PDOStatementProxy.php
appwrite                        | [Error] Line: 64
```

@Meldiron, would you please double check?

**Comment by andrewmoore-nz** [comment]
I can confirm, getting consistent behaviour on this bug, across multiple collections. Only seems to be an issue when querying for that particular attribute e.g. I can query for other non-hyphenated attributes in the same collection and all behaves normally.

**Comment by stnguyen90** [comment]
@abnegate this isn't in appwrite yet, right? We need a utopia-php/database release and a bump of the version in appwrite?

**Comment by abnegate** [comment]
@stnguyen90 Correct, it won't be included in 1.2.1 as the database release includes other changes we don't want in Appwrite yet 👍 these are getting auto-closed by the PR link
